android/samples/sample-gltf-viewer/src/main/java/com/google/android/filament/gltf/MainActivity.kt

```diff
@@ -80,9 +80,8 @@ class MainActivity : Activity() {
             input.read(bytes)
             ByteBuffer.wrap(bytes)
         }
-        modelViewer.loadModelGltf(buffer) { uri ->
-            readCompressedAsset("models/$uri")
-        }
+
+        modelViewer.loadModelGltfAsync(buffer, { uri -> readCompressedAsset("models/$uri") })
```

Comment:
You don't necessarily need to use the `Async` suffix since coroutine methods are marked in the IDE.

Reply:
Well I need to disambiguate it from its Sync counterpart, which I think should still exist for simple use cases.